guides/operator/custom-job-stages/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: single
+title:  "Custom Job Stages"
+sidebar:
+  nav: guides
+---
+
+Much like [Custom Webhook Stages](/guides/operator/custom-webhook-stages/), Custom Job Stages enable Spinnaker operators to extend Spinnaker without having to maintain a custom build of Orca. For users of Kubernetes or Titus, Custom Job Stages offer a native solution for enhancing Spinnaker with functionality with the power of containers.
+
+*Note: Custom Job stages are only supported for the Kubernetes and Titus providers.*
+
+## How it works
+
+At it's core, Custom Job Stages use the Run Job stage to start a Task (Titus) or Job (Kubernetes) to perform some type of work. These jobs are defined as configuration for Orca and registered when the application starts. When used in a pipeline, Orca takes this configuration, generates a Run Job stage and then exeucutes it. To support a more native experience, we provide configuration options for things like stage name, description and parameters so that this stage feels less like a Run Job stage and more like an out of the box stage. 
```

Comment:
```suggestion
At its core, Custom Job Stages use the Run Job stage to start a Task (Titus) or Job (Kubernetes) to perform some type of work. These jobs are defined as configuration for Orca and registered when the application starts. When used in a pipeline, Orca takes this configuration, generates a Run Job stage and then exeucutes it. To support a more native experience, we provide configuration options for things like stage name, description and parameters so that this stage feels less like a Run Job stage and more like an out of the box stage. 
```

Reply:
it's --> its